doc/release-notes.md

```diff
@@ -64,6 +64,12 @@ P2P and network changes
 Updated RPCs
 ------------
 
+- The `-deprecatedrpc=addresses` flag has been removed entirely.
+  The following RPCs:  `gettxout`, `getrawtransaction`, `decoderawtransaction`,
+  `decodescript`, `gettransaction`, and REST endpoints: `/rest/tx`, `/rest/getutxos`,
+  `/rest/block` no longer return the fields: `addresses`, `reqSigs` which were
+  previously deprecated in v22. (#22650)
```

Comment:
IIUC gettransaction is only affected when verbose=true and some writing suggestions
```diff
-- The `-deprecatedrpc=addresses` flag has been removed entirely.
-  The following RPCs:  `gettxout`, `getrawtransaction`, `decoderawtransaction`,
-  `decodescript`, `gettransaction`, and REST endpoints: `/rest/tx`, `/rest/getutxos`,
-  `/rest/block` no longer return the fields: `addresses`, `reqSigs` which were
-  previously deprecated in v22. (#22650)
+- The `-deprecatedrpc=addresses` configuration option has been removed.  RPCs
+  `gettxout`, `getrawtransaction`, `decoderawtransaction`, `decodescript`,
+  `gettransaction verbose=true` and REST endpoints `/rest/tx`, `/rest/getutxos`,
+  `/rest/block` no longer return the `addresses` and `reqSigs` fields, which
+  were previously deprecated in 22.0. (#22650)
```